support/run/test_template.ejs

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
     <title></title>
-    <script src="skulpt.js" type="text/javascript"></script>
+    <script src="skulpt.min.js" type="text/javascript"></script>
```

Comment:
is this change on purpose? I thought we used the dev build for browser debugging?

Reply:
I needed to test that changing the output language produces a distribution file that still functions in a web browser.  As I thought about it, I started to think that we should be testing on the "real" distribution file under normal circumstances, as well.  The "brun" command still uses the dev build, as that seems more appropriate.  But, what does it matter if all the tests pass on the dev build?  They need to pass on the real build...

Ultimately, I think this is debatable and depends on what one is trying to do at the time.  Perhaps there needs to be a switch to allow either somehow?  Or I can switch it back.

Reply:
A future update to this might be detecting which (minified or non minified) is available and adding that to the template.

Reply:
Easily done, so I did.